binr/radare2/radare2.c

```diff
@@ -874,7 +874,11 @@ int main(int argc, char **argv, char **envp) {
 		r_config_set (r.config, "scr.interactive", "false");
 		r_config_set (r.config, "scr.prompt", "false");
 	}
-
+#if __UNIX__
+	if (!r_cons_isatty ()) {
```

Comment:
This is too late to check this setting since when using ```-qc "pxw 32"``` it bails out on line 862.